nubis/puppet/consul.pp

```diff
@@ -29,21 +29,21 @@
 
 # XXX: need to move to puppet-envconsul proper
 staging::file { 'envconsul.tar.gz':
-  source => "https://www.github.com/hashicorp/envconsul/releases/download/v0.5.0/envconsul_0.5.0_linux_amd64.tar.gz",
+  source => "https://www.github.com/hashicorp/envconsul/releases/download/v0.6.0/envconsul_0.6.0_linux_amd64.tar.gz",
```

Comment:
It looks like they changed it from `.tar.gz` to `.zip` will get a fix in for this